### PR TITLE
Update for SDK 22.11

### PR DESCRIPTION
**Pull request by kevinstratford**
With CUDA 11.8

### CODE REVIEW

**Comment by mbareford** [comment]
In the "Compiling and using GPU-aware MPI" section, why isn't there a `4.1.4-cuda-11.8-nvfortran` module?
I think a reader would wonder why cuda 11.6 is suddenly being used.

**Comment by aturner-epcc** [comment]
@mbareford A quick test shows that `openmpi/4.1.4-cuda-11.8` has been compiled with `nvfortran` support so there is no need for a separate `openmpi` module:

```
[aturner@cirrus-login1 fortran_cuda_support]$ module load nvidia/nvhpc/22.11 
Loading nvidia/nvhpc/22.11
  Loading requirement: gcc/8.2.0           
[aturner@cirrus-login1 fortran_cuda_support]$ module load openmpi/4.1.4-cuda-11.8
[aturner@cirrus-login1 fortran_cuda_support]$ mpif90 --version

nvfortran 22.11-0 64-bit target on x86-64 Linux -tp haswell 
NVIDIA Compilers and Tools
Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

File: user-guide/gpu.rst [hunk 1]
```diff
@@ -539,11 +537,18 @@ MPI calls. This allows replacement of relevant host device transfers by
...
 ::
 
+   $ module load openmpi/4.1.4-cuda-11.8
+   $ module load nvidia/nvhpc-nompi/22.11
+
+and for Fortran use
+
+::
+   
    $ module load openmpi/4.1.4-cuda-11.6-nvfortran
```
Comment (aturner-epcc) [inline user-guide/gpu.rst:552]: This needs to be changed to:

```
module load openmpi/4.1.4-cuda-11.8
```

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
One minor change to a module version

**Comment by mbareford** [comment]
LGTM

**Pull request merged**